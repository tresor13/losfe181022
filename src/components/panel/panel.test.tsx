import renderer from 'react-test-renderer';
import Panel from './index';

it('Panel should render', () => {
  const component = renderer.create(<Panel image="test"/>);
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});


it('Image prop should be correct', () => {
  const component = renderer.create(<Panel image="test"/>);
  expect(component.root.props?.image).toEqual('test');
});
