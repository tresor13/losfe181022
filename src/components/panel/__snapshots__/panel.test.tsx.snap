// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Panel should render 1`] = `
<section
  className="section"
  style={
    Object {
      "backgroundImage": "url(test)",
    }
  }
/>
`;
